src/main/java/seedu/address/model/person/Year.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's Year in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidYear(String)}
+ */
+public class Year {
+    public static final String MESSAGE_CONSTRAINTS = "Year should be a positive number";
+    public static final String VALIDATION_REGEX = "[1-9]\\d*";
+    public final String value;
+
+    /**
+     * Constructs a {@code Year}.
+     *
+     * @param year A valid year.
+     */
+    public Year(String year) {
+        requireNonNull(year);
```

Comment:
Do you account for handling the null pointer exception thrown possibly? Perhaps can add assertion statement also.